ctgan/data_transformer.py

```diff
@@ -18,8 +18,8 @@
 class DataTransformer(object):
     """Data Transformer.
 
-    Model continuous columns with a BayesianGMM and normalized to a scalar [0, 1] and a vector.
-    Discrete columns are encoded using a scikit-learn OneHotEncoder.
+    Model continuous columns with a BayesianGMM and normalizes them to a scalar between [-1, 1]
+    and a vector. Discrete columns are encoded using a scikit-learn's OneHotEncoder.
```

Comment:
minor: either leave as was or get rid of the 'a' before scikit

Reply:
Both are wrong actually, we use RDT's OneHotEncoder...